resources/world_gen.py

```diff
@@ -374,12 +476,16 @@ def surface_builder(rm: ResourceManager, name: str, surface_builder):
     }))
 
 
-def forest_config(min_rain: float, max_rain: float, min_temp: float, max_temp: float, tree: str, old_growth: bool):
+def forest_config(min_rain: float, max_rain: float, min_temp: float, max_temp: float, log: str, leaves: str, twig: str, fallen_leaves: str, tree: str, old_growth: bool):
     cfg = {
         'min_rain': min_rain,
         'max_rain': max_rain,
         'min_temp': min_temp,
         'max_temp': max_temp,
+        'log': log,
+        'leaves': leaves,
+        'twig': twig,
+        'fallen_leaves': fallen_leaves,
```

Comment:
This should be able to just string replace here with `tree`, i.e. `'tfc:fallen_leaves/%s' % tree`